src/app.rs

```diff
@@ -72,6 +72,12 @@ pub fn build() -> App<'static, 'static> {
                 .long("ignore-config")
                 .help("Ignore the configuration file"),
         )
+        .arg(
+            Arg::with_name("config-file")
+                .short("f")
```

Comment:
`-f` is taken by `GNU ls`:

> -f     do not sort, enable -aU, disable -ls --color

Reply:
I think just `--config-file` would be enough.

Reply:
Ok, will do.